proposals/Addresses/qa.json

```diff
@@ -130,5 +130,17 @@
         "addr": "0x14d9548b384dfd5612c5E421584645aE9a3eef38",
         "chainId": 421614,
         "isContract": true
+    },
+    {
+        "name": "DEPLOYER",
+        "addr": "0x5Ec41e3a9c712D0BBC26d2CbA0E653c5d2cc982C",
+        "chainId": 421614,
+        "isContract": false
+    },
+    {
+        "name": "AFTER_DEPLOY_CALLER",
```

Comment:
No longer needed after this PR https://github.com/ZTX-Foundation/tuxedo/pull/233

Reply:
Thanks @prateek105. Fixed.